scripts/auth0.js

```diff
@@ -86,6 +85,7 @@ window.auth0.createAuth0Client({
       if (user?.email?.toLowerCase() === 'self.service.test.user@gmail.com') {
         document.body.classList.add('is-test-user');
       }
+      document.dispatchEvent(new CustomEvent('auth0:authenticated'));
```

Comment:
Just curious what this change is fixing ?

Reply:
preventing race condition. There was a chance is-headwire was not added yet, so the admin tab would not appear because it was checked for too early.